src/QueueManager.php

```diff
@@ -98,6 +98,26 @@ public static function setConfig($key, $config = null): void
             throw new BadMethodCallException('Must specify `url` key.');
         }
 
+        if (!empty($config['queue'])) {
+            if (!is_array($config['url'])) {
+                $config['url'] = [
+                    'transport' => $config['url'],
+                    'client' => [
+                        'router_topic' => $config['queue'],
+                        'router_queue' => $config['queue'],
+                        'default_queue' => $config['queue'],
+                    ],
+                ];
+            } else {
+                $clientConfig = $config['url']['client'] ?? [];
+                $config['url']['client'] = $clientConfig + [
+                    'router_topic' => $config['queue'],
+                    'router_queue' => $config['queue'],
+                    'default_queue' => $config['queue'],
+                ];
+            }
+        }
```

Comment:
Should we also have a test that uses `engine()` to read configuration from the created client?